mlflow/tracking/client.py

```diff
@@ -483,6 +483,12 @@ def get_trace(self, request_id: str, display=True) -> Trace:
             request_id = "12345678"
             trace = client.get_trace(request_id)
         """
+        if is_databricks_uri(self.tracking_uri) and request_id.startswith(TRACE_REQUEST_ID_PREFIX):
```

Comment:
Can we verify that the trace looks like a UUID?

Reply:
/ai Write a function to check if the given string looks like a UUID